demos/browser/app/meeting/meeting.ts

```diff
@@ -474,6 +475,32 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
     });
   }
 
+  getNearestMediaRegion(): void {
+    const defaultNearestMediaRegion = "us-east-1";
+    new AsyncScheduler().start(
+      async (): Promise<void> => {
+        this.showProgress('progress-authenticate');
+        try {
+          const nearestMediaRegionResponse = await fetch(
+            `https://nearest-media-region.l.chime.aws`,
+            {
+              method: 'GET',
+            }
+          );
+          const nearestMediaRegionJSON = await nearestMediaRegionResponse.json();
```

Comment:
the join button event handler created in initEventListeners is also an aysnc job which reads from inputRegion. So if this job for some reason completes after the users click Join, it would not work. Maybe its fine and we can just use the default region in that case. What do you think @simmkyu